### PR TITLE
feat(gui): Add ref params to homepage links in menu

### PR DESCRIPTION
**Pull request by jhermsmeier**
This adds a `ref` param to the URLs in the menu,
in order to see where page views are coming from.

Change-Type: patch

### CODE REVIEW

File: lib/gui/menu.js [hunk 1]
```diff
@@ -84,13 +84,13 @@ const buildWindowMenu = (window) => {
         {
           label: 'Etcher Pro',
           click () {
-            electron.shell.openExternal('https://etcher.io/pro')
+            electron.shell.openExternal('https://etcher.io/pro?ref=etcher_menu')
```
Comment (jhermsmeier) [inline lib/gui/menu.js]: @tzikis what would be preferable here? `utm_source=etcher_menu`?

**Comment by lurch** [comment]
See also https://github.com/resin-io/etcher-homepage/pull/89

File: lib/gui/menu.js (hunk 1 above)
Comment (tzikis) [reply lib/gui/menu.js]: @jhermsmeier as discussed in flowdock, i need `?utm_source=` for Mailchimp, but we can keep `?ref=` if we wanna use it on mixpanel

Comment (jhermsmeier) [reply lib/gui/menu.js]: @tzikis aye, I've added `utm_source` now, while also keeping the `ref` for Etcher Pro

**Comment by lurch** [comment]
Looks like the lint tests are causing the CI builds to fail?
```
node scripts/html-lint.js
Scanning...
3 errors at lib/gui/app/index.html
  [17:69] An element with the attribute “tabindex” must not appear as a descendant of the “button” element.
  [24:59] An element with the attribute “tabindex” must not appear as a descendant of the “button” element.
  [31:68] An element with the attribute “tabindex” must not appear as a descendant of the “button” element.
Total: 7/8
make: *** [lint-html] Error 1
```

**Pull request merged**